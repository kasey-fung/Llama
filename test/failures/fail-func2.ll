void printTwo(int a, int b):
    print(a)
    print(b)

void printTwo(int x, int y):
    print(x)
    print(y)

void main():
    printTwo(1, 2)
