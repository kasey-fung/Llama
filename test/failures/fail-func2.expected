Fatal error: exception Failure("duplicate function printTwo")